### ISSUE TITLE
Put klass into block  for `update_enumerations_model`

### ISSUE THREAD

**Issue by zekefast** [albertosaurus/power_enum#13]
Then instead such call

``` ruby

BookingStatus.update_enumerations_model do
  BookingStatus.create :name        => 'Foo',
                       :description => 'Bar',
                       :active      => false
end

```

It would be possible to make call like this

``` ruby

BookingStatus.update_enumerations_model do |klass|
  [....].each do
    klass.create(....)
  end
end
```

**Comment by albertosaurus** [comment]
I'm not opposed to this per se, but what does it actually buy you?

**Comment by albertosaurus** [comment]
Now that I think about it, there are some cases where it may make sense.

**Comment by albertosaurus** [comment]
Implemented in 0.9.3.

**Comment by zekefast** [comment]
Thanks!
